fca/src/main/java/org/nmdp/ngs/fca/Concept.java

```diff
@@ -46,10 +41,17 @@ public Concept(final MutableBitSet extent, final MutableBitSet intent) {
         this.extent = extent;
         this.intent = intent;
     }
+    
+    private MutableBitSet and(final Concept that) {
+        return new MutableBitSet().or(this.intent).and(that.intent);
```

Comment:
Would it be better to add to dsh-bitset a copy constructor
```java
return new MutableBitSet(this.intent).and(that.intent);
```
or add a new method ```mutableCopy()```?
```java
return this.intent.mutableCopy().and(that.intent);
```